src/renderer/app-paths/app-paths.injectable.ts

```diff
@@ -2,15 +2,17 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { getInjectable } from "@ogre-tools/injectable";
 import { AppPaths, appPathsInjectionToken, appPathsIpcChannel } from "../../common/app-paths/app-path-injection-token";
 import getValueFromRegisteredChannelInjectable from "./get-value-from-registered-channel/get-value-from-registered-channel.injectable";
 
 let syncAppPaths: AppPaths;
 
 const appPathsInjectable = getInjectable({
+  id: "app-paths",
+
   setup: async (di) => {
-    const getValueFromRegisteredChannel = di.inject(
+    const getValueFromRegisteredChannel = await di.inject(
```

Comment:
Notice: `di.inject` which happens in `setup` is always async because if the injected injectable is also setuppable, the setup will be ran first asynchronously before instantiation.